src/components/field-label/field-label.js

```diff
@@ -33,13 +33,16 @@ export const FieldLabel = props => (
       </Spacings.Inline>
 
       {props.hint && (
-        <Spacings.Inline alignItems="center" scale="xs">
-          {props.hintIcon &&
-            // FIXME: add proper tone when tones are refactored
-            React.cloneElement(props.hintIcon, {
-              size: 'medium',
-              theme: props.hintIcon.props.theme || 'orange',
-            })}
+        <Spacings.Inline alignItems="center" scale="s">
+          {props.hintIcon && (
+            <div>
+              {React.cloneElement(props.hintIcon, {
+                // FIXME: add proper tone when tones are refactored
+                size: props.hintIcon.props.size,
```

Comment:
Default size to be `medium`? Otherwise it's a breaking change - ain't it?

Reply:
Oh yeah, and Percy caught that as well 😄